File: Java/Matkul Algoritma (Error Semua)/Kumpulan kodingan java/37 kodingan pra uts/tahunkabisat.java
import java.util.Scanner;

public class tahunkabisat{

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		
		int tahun;
		System.out.print("masukkan Tahun: ");
		tahun = scan.nextInt();
		if(tahun%4==0)
			System.out.print("tahun "+tahun+" merupakan tahun kabisat");
		else
			System.out.print("tahun "+tahun+" bukan merupakan tahun kabisat");
	}

}

File: C++/Praktikum Dasar Pemrograman/modul 2 Operator, Logika, If/tahun kabisat.cpp
#include<iostream>
using namespace std;

int a;
// a = tahun
int main()
{
	cout << "PROGRAM PENDETEKSI TAHUN KABISAT" << endl;
	cout << "--------------------------------" << endl;
	
	cout << "Masukan tahun : ";
	cin >> a;
	if(a%4==0){
		cout << "tahun tersebut merupakan tahun KABISAT";
	}
	else {
		cout << "bukan tahun KABISAT";
	}

}
